### PR TITLE
Create option for api with admin

### PR DESCRIPTION
**Pull request by Ru-T**
## Why?
- For apps that are api but with an admin interface
## What Changed?
- Created option for api with admin interface
- If this option is chosen, do not delete front end files or edit application controller
- Otherwise, the same as api_only install

### CODE REVIEW

File: template.rb [hunk 1]
```diff
@@ -186,17 +202,17 @@ def install_optional_gems
...
+    api_only_install
+  end
 else
   integrated_app_install
-  devise?
-  active_admin?
-  cucumber_capybara?
 end
 smashing_docs?
-
```
Comment (jahammo2) [inline template.rb:199]: I really like the separation of responsibility throughout these methods.

**Comment by jahammo2** [comment]
This looks great so far but maybe one more set of :eyes: from someone else who's familiar with this project.

**Comment by TylerRockwell** [comment]
Looks great to me @Ru-T :shipit:

**Comment by Ru-T** [comment]
Thanks @jahammo2 and @TylerRockwell!

**Pull request merged**